code/game/objects/items.dm

```diff
@@ -832,48 +832,48 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 		var/timedelay = usr.client.prefs.tip_delay/100
 		var/user = usr
 		tip_timer = addtimer(CALLBACK(src, .proc/openTip, location, control, params, user), timedelay, TIMER_STOPPABLE)//timer takes delay in deciseconds, but the pref is in milliseconds. dividing by 100 converts it.
-		var/mob/living/L = usr
-		if(istype(L) && L.incapacitated())
-			apply_outline(COLOR_RED_GRAY)
+		var/mob/living/alive = usr
```

Comment:
This is still a bad variable name, it doesn't describe the context.

Just name it `user` or `living_user`.